### PR TITLE
Add reading time for posts in iOS

### PR DESCRIPTION
**Pull request by dfernandezm**
## What does this pull request change?

Add reading time for posts to iOS, same as Android.

## Screenshots

Screenshot 2022-04-11 at 07 36 17

## How is this change tested?

Manually, until more focus is put into UI tests for Kotlin/Swift

### CODE REVIEW

File: appIos/appIos.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -344,6 +344,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = 5G893F5HW7;
 				ENABLE_PREVIEWS = YES;
+				"EXCLUDED_ARCHS[sdk=*]" = "";
```
Comment (dfernandezm) [inline appIos/appIos.xcodeproj/project.pbxproj:347]: I'm not sure why has this been added, can remove it

File: appIos/post/post/PostScreen.swift [hunk 2]
```diff
@@ -179,7 +183,7 @@ private struct PostAuthor: View {
                     font: theme.typography.subtitle
                 )
                 AppText(
-                    text: "1 min read",
+                    text: String(format: "%d min read", readingTimeMinutes),
```
Comment (dfernandezm) [inline appIos/post/post/PostScreen.swift:186]: I haven't checked the equivalent to the `strings` resources in Android. I guess that's the proper way to format this.

File: appIos/post/post/PostScreen.swift [hunk 3]
```diff
@@ -69,7 +69,9 @@ private struct PostState: View {
                             alignment: .leading,
                             spacing: 8
                         ) {
-                            PostHeader(header: post.raw.headline ?? "--")
+                            PostHeader(
+                                header: post.raw.headline ?? "--",
```
Comment (gchristov) [inline appIos/post/post/PostScreen.swift:73]: Is this cast necessary here?

File: appIos/post/post/PostScreen.swift (hunk 2 above)
Comment (gchristov) [reply appIos/post/post/PostScreen.swift:186]: Nah it’s okay for now - there is a way but the project doesn’t support yet which is a separate piece of work.

File: appIos/appIos.xcodeproj/project.pbxproj (hunk 1 above)
Comment (gchristov) [reply appIos/appIos.xcodeproj/project.pbxproj:347]: Not sure what it means, will check…

**Review by gchristov — COMMENTED** [review]
Nice! I’d like to check that new property that got added by the M1 I imagine.

Otherwise my inly comment was around the Int cast which I wasn’t sure about if it’s needed.

File: appIos/post/post/PostScreen.swift (hunk 3 above)
Comment (dfernandezm) [reply appIos/post/post/PostScreen.swift:73]: This is Xcode complaining that `KotlinInt` cannot be passed as `Int`
Screenshot 2022-04-11 at 07 49 28

Comment (dfernandezm) [reply appIos/post/post/PostScreen.swift:73]: Another posh Apple issue? :)

File: appIos/appIos.xcodeproj/project.pbxproj (hunk 1 above)
Comment (gchristov) [reply appIos/appIos.xcodeproj/project.pbxproj:347]: I don't think this should be here, interesting why Xcode added it - lets revert it from git

File: appIos/post/post/PostScreen.swift (hunk 3 above)
Comment (dfernandezm) [reply appIos/post/post/PostScreen.swift:73]: I see, these 2 guys copying each other with the nullability. I'll apply this one

File: appIos/appIos.xcodeproj/project.pbxproj (hunk 1 above)
Comment (dfernandezm) [reply appIos/appIos.xcodeproj/project.pbxproj]: Removed

**Pull request merged**